app/views/devise/mailer/reset_password_instructions.html.erb

```diff
@@ -1,19 +1,21 @@
-<p>
-  <%= t('.greetings', name: @resource.name) %>
+<p class="h1">
+  <%= t('.greetings',
+        name: @resource.name.present? ? name : t('.no_name_fallback')) %>
```

Comment:
This is not going to work, as `name` will be an unknown variable. It should be
```
<%= t('.greetings', 
      name: @resource.name.present? ? @resource.name : t('.no_name_fallback')) %>
```

Reply:
I just realized you can make it even shorter by using the `#presence` method of Rails:
```
<%= t('.greetings', 
      name: @resource.name.presence || t('.no_name_fallback')) %>
```

Reply:
Thanks, imo the syntax of this is a bit weird, but that seems like the way to go.
I verified that the name is displayed when existing (after adding the `@resource.` prefix ofc)